zeq/vocabulary.h

```diff
@@ -59,7 +61,22 @@ ZEQ_API Event serializeRequest( const uint128_t& eventType );
  * @return an uint128_t to identify the zeq event that should be published.
  */
 ZEQ_API uint128_t deserializeRequest( const Event& event );
+//@}
 
+/**
+ * @name JSON/binary event translation.
+ *
+ * These functions are not thread-safe, that is, calling registerEvent()
```

Comment:
You call that function in user code? Where and why? Usually, flatbuffers generated calls that on init, so I'm wondering what use case I'm missing.